### DISCUSSION TITLE
DAE Scipy solver errors

### DISCUSSION THREAD

**Discussion by jacoblufc** [community/community#124108] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Using the scipy package  DAE-Scipy https://github.com/laurent90git/DAE-Scipy to solve an DAE I keep getting message: Required step size is less than spacing between numbers.
  success: False, error in numerics of the solver. Tried adjusted dt and discrete step size..

**Comment by jacoblufc** [comment]
@laurent90git

**Comment by laurent90git** [comment]
Without a minimal example that I can use to reproduce the issue, there's not much I can do !

**Reply by jacoblufc** [reply]
ofc sorry! I just invited you to a repo where I have put my script that uses radauDAE. So my system is a discretised beam is 16 dim when N=3 3 links

**Comment by ebndev** [comment]
Hi @jacoblufc,

Thanks for being a part of the GitHub Community, we're glad you're here!

It looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place. As this is a public forum, I will be locking this discussion.

Best of luck!
